src/main/java/model/CarRacingGame.java

```diff
@@ -0,0 +1,33 @@
+package model;
+
+import java.util.List;
+
+public class CarRacingGame {
+    private TryCount tryCount;
+    Cars cars;
+
+
+    // Cars를 가지고 있는 것이 어떨까?
+    public CarRacingGame(int tryCount, String carNames) {
+        this.tryCount = new TryCount(tryCount);
+        this.cars = new Cars(carNames);
+    }
```

Comment:
```suggestion
    public CarRacingGame(int tryCount, String carNames) {
       this(new TryCount(tryCount), new Cars(carNames));
    }
    
    public CarRacingGame(TryCount tryCount, Cars cars) {
        this.tryCount = tryCount;
        this.cars = cars;
Comment view
    }
```
객체의 필드 타입과 같은 주 생성자를 추가하고, 다른 타입을 가지는 부 생성자는 주 생성자를 호출해 값을 초기화한다.
어제 진행한 4차 라이브 강의의 "28분 13초: 다수의 생성자의 필요성, 주 생성자 & 부 생성자" 영상을 참고한다.